### DISCUSSION TITLE
Dedicated Pet Spawns

### DISCUSSION THREAD

**Discussion by Jav1k** [FakeFishGames/Barotrauma#6315] [Ideas and suggestions]
Since pets seem to spawn randomly at any human spawn points, every time the sub docks to a station or leaves from one, all the pets will be strewn around the sub. On a large sub this would mean players have to run around and collect them so they dont starve, forgotten in some ballast tank, which, depending on the number of pets and sub size will take several minutes each mission and gets extremely tedious very quickly. Therefore I suggest adding a new option to the human spawn "assigned jobs" for pets, or a new type of spawn altogether, or have their location be carried over from wherever they were when loading into a new mission, as this would make it possible to have a dedicated pet room.

**Comment by Ek-Videogames** [comment]
I'm not sure if this 'bug' is still in, but pets would also include spawnpoints **on linked shuttles** as valid spawn locations - unlike humans. So pets would occasionally spawn inside a submarine's drone / shuttle.
